main.go

```diff
@@ -177,6 +181,10 @@ func action(f func(*dbmate.DB, *cli.Context) error) cli.ActionFunc {
 		db.MigrationsDir = c.GlobalString("migrations-dir")
 		db.SchemaFile = c.GlobalString("schema-file")
 		db.WaitBefore = c.GlobalBool("wait")
+		overrideTimeout := c.GlobalDuration("wait-timeout")
+		if overrideTimeout != 0 {
+			db.WaitInterval = overrideTimeout
```

Comment:
pretty sure you want to set `db.WaitTimeout` here, not `db.WaitInterval`

Reply:
Thank you, I going to fix it